spec/requests/moderations_spec.rb

```diff
@@ -135,4 +136,52 @@
       expect(response.body).to include "circle centered-icon adjustment-icon subtract"
     end
   end
+
+  describe "/mod" do
+    # rubocop:disable Layout/LineLength
+    let(:coc_guides_copy) { 'Check out our <a href="/code-of-conduct">Code of Conduct</a> and read through our <a href="/community-moderation">Trusted User Guide</a> and <a href="/tag-moderation">Tag Moderation Guide</a>.' }
+    # rubocop:enable Layout/LineLength
+    let(:become_mod_copy) { "If you'd like to assist us as a trusted user or tag mod" }
+    let(:logged_out_copy) { "P.S. You are not currently signed in." }
+
+    context "when on dev.to" do
```

Comment:
Hm, now that we are not using `dev_to?`, do we need 2 different context for these tests? Seems like the behavior should be the same regardless of what Forem you are on, so we can consolidate these I think :)

Reply:
@vaidehijoshi Alright I can do that as you say.

I wrote the tests this way because there is a line in the copy that references `community_name` and i thought it a good idea to test this for different communities 🤷🏾 

`app/views/moderations/index.html.erb:39`

```erb
<p>We periodically award some <%= community_name %> members with heightened privileges to help moderate the community.</p>
```
But I'll take it out.

Reply:
@msarit Thanks for the additional context! 

I wanted to explain a little bit about _why_ I am suggesting this. In the view where you added these changes, you're using the `community_name` helper, which is defined in the `ApplicationHelper`. 

https://github.com/forem/forem/blob/3dd42fe8a0ebc00df619fd2d3a6a44b125cba18d/app/views/moderations/index.html.erb#L36

While the view is using the helper, there isn't any _other_ logic happening here. Previously, when you were checking `SiteConfig.dev_to?`, you _did_ have some conditional rendering logic, but after you updated this PR, you're using the value of that helper directly.

The helper itself is already tested in the `application_helper_spec.rb` (admittedly, there's only one test for this):
https://github.com/forem/forem/blob/8d462801ece4a866a4aaf4a3844759e23d88d866/spec/helpers/application_helper_spec.rb#L39-L44

Looking at the view, the only place where you might encounter different behavior is the line that renders based on whether or not the user is signed in or not; the `community_name` helper's behavior doesn't really have anything to do with the view, it is its own method. So when you test against the two different community names on these lines:
https://github.com/forem/forem/blob/3dd42fe8a0ebc00df619fd2d3a6a44b125cba18d/spec/requests/moderations_spec.rb#L149
https://github.com/forem/forem/blob/3dd42fe8a0ebc00df619fd2d3a6a44b125cba18d/spec/requests/moderations_spec.rb#L174

...you're basically testing the `community_name` helper, somewhat inadvertently (if that makes sense). In other words, the behavior you are testing (setting the `community_name` to `"DEV"` and then to `"Forem"`) isn't behavior that has anything to do with this _view_ -- it is something that the helper handles :)

Turning our attention back to the view, the things that you probably care about testing are that
1) the links render and have the right copy and
2) you have accounted for this conditional logic:
https://github.com/forem/forem/blob/3dd42fe8a0ebc00df619fd2d3a6a44b125cba18d/app/views/moderations/index.html.erb#L39-L41

The good news is that you already have tests for all of this :) 
https://github.com/forem/forem/blob/3dd42fe8a0ebc00df619fd2d3a6a44b125cba18d/spec/requests/moderations_spec.rb#L155-L169

I think from reading the tests, someone might expect the view to _behave_ differently depending on whether it was being rendered on the DEV forem or some other forem instance. But the view won't really behave differently -- the only thing that will change will be the `community_name`, and as we saw, that is already tested 😎 

So all of this is to say, I think you don't need to test against the "dev.to" scenario versus the "other forem" scenario for this, since the behavior of the _view_ doesn't change in any case. 😃 Back when you were checking `SiteConfig.dev_to?`, this would have made perfect sense, but now that we don't have that check anymore, the behavior of the view really is the same across the board.

I hope that this additional context makes more sense. As always, I'm more than happy to give more explanation or context if you ever want it! I never want anyone to feel like they have to make a change "just because I said so"; I care a whole lot more about explaining where I'm coming from and sharing the _why_ so that we can both learn from each other :) If you ever want me to give a more in-depth explanation, I'm 10000% down to take the time to write it out!

Reply:
Ok this makes sense then! Thank you! I will complete the changes tomorrow 🙏🏽